R/Optimizer.R

```diff
@@ -11,6 +11,12 @@
 #' of the [OptimInstance] at the end in order to store the best point  and its
 #' estimated performance vector.
 #'
+#' @section Progress Bars:
+#' `$optimize()` supports progress bars via the package \CRANpkg{progressr}
+#' combined with [Terminator] subclasses with the `progressr` property. Simply
```

Comment:
As we encapsulated the `progressr` package, we could call it just `progress` property?